docs/index.md

```diff
@@ -15,46 +15,50 @@ In order to integrate with Lead Safe, the health provider will need to make a RE
+| visit_date      | Date     | yyyy-mm-dd                         | Effective date of score                                                                                |
+| network_id      | Text     | <N/A>                              | The parent entity of the submitting organization                                                                  |
+| clinic_id       | Text     | <N/A>                              | Organization ID                                                                                                   |
+| location_id     | Text     | <N/A>                              | Specific Location Abbreviation in Centricity                                                                      |
+| patient_id      | Text     | <N/A>                              | AlphaNumeric patient identifier                                                                                                |
+| address1        | Text     | <N/A>                              | Patient home address (street number, street direction, street name, street type)                                  |
+| address2        | Text     | <N/A>                              | Patient home address (additional information, e.g. apartment number)                                              |
+| city            | Text     | <N/A>                              | Patient home address city                                                                                         |
+| state           | Text     | <N/A>                              | Patient home address state                                                                                        |
+| zip             | Text     | <N/A>                              | Patient home address zip code                                                                                     |
+| date_of_birth   | Date     | yyyy-mm-dd                         | Patient date of birth                                                                                             |
+| gender          | Text     | M/F/U                              | Patient gender                                                                                                    |
+| race            | Text     | Code Values from Centricity        | Standard ONC Race definitions <br> <br> **Code / description** <br> 1002-5 / American Indian or Alaska Native <br> 2028-9 / Asian <br> 2054-5 / Black or African American <br> 2076-8 / Native Hawaiian or Other Pacific Islander <br> 2106-3 / White <br> UNK / Unknown |
+| ethnicity       | Text     | Code Values from Centricity        | Standard ONC Ethnicity definitions <br> <br> **Code / description** <br> 2135-2 / Hispanic or Latino <br> 2186-5 / Non Hispanic or Latino <br> UNK / Unknown |
+| VISIT ARRAY     |          |                                    |                                                                                                                   |
```

Comment:
I would clarify that this array may be empty. Should it be optional? In that case, missing means no visits. Ditto for the lab array.

Reply:
Ok, this makes sense. We'll update to specify that it may be empty.

Reply:
@potash - from the perspective of the runner, which would you prefer to have: a `visit` array that is blank, i.e., `"visit": [ ]` or to just omit it overall? I'm assuming the latter.

Reply:
Per the discussion today, we will leave the entire array as optional (will do the same for `lab` array, too.